playbooks/templates/tigkstack/telegraf.conf.j2

```diff
@@ -39,6 +39,12 @@
   write_consistency = "any"
   timeout = "30s"
 
+{% if outputs_prometheus_client is defined %}
+[[outputs.prometheus_client]]
+  listen = "{{ outputs_prometheus_client_listen|default(':9126') }}"
```

Comment:
the variable should also follow the same prefix ``telegraf_outputs_prometheus_client_listen``.